include/logger/err.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef ERROR_HPP
+#define ERROR_HPP
+#include <string>
+#include <map>
+
+/*
+ *@brief namespace for logger lib .
+ */
+namespace logger {
+
+/*
+ *@brief namespace for error code .
+ */
+namespace error_code {
+
+  const std::string ERROR_101{ "[Error 101] : Data found to be empty." };
```

Comment:
error string cant be error code 

Check this, Try to do it like this.
https://github.com/ifm/ifm3d/blob/master/modules/camera/src/libifm3d_camera/err.cpp